src/apiProxy/api-proxy.js

```diff
@@ -89,13 +89,22 @@ export const parseApiResponse = ([response, body]) => {
  * @param {Object} query a query object from the application
  * @return {Object} the arguments for api request, including endpoint
  */
-export function queryToApiConfig({ type, params, flags }) {
-	const configCreator = apiConfigCreators[type];
-	if (!configCreator) {
-		throw new ReferenceError(`No API specified for query type ${type}`);
+export function queryToApiConfig({ endpoint, type, params, flags }) {
+	if (!endpoint) {
```

Comment:
can we log `type` or `ref` so we know which api needs updating? Also, I wouldn't mind a ToDo to remove this code in future minor versions?